apps/issuer/serializers_v1.py

```diff
@@ -224,6 +226,44 @@ class BadgeClassExpirationSerializerV1(serializers.Serializer):
                                        choices=BadgeClass.EXPIRES_DURATION_CHOICES)
 
 
+class SuperBadgeBadgeSerializerV1(serializers.ModelSerializer):
+    id = serializers.RelatedField(queryset=BadgeClass.objects.all())
+    superBadge = serializers.RelatedField(queryset=SuperBadge.objects.all(),
+            write_only=True, required=False)
+
+    class Meta:
+        model = SuperBadge
+        # list_serializer_class = SuperBadgeBadgeSerializerV1
+        fields = '__all__'
+        # apispec_definition = ('CollectionBadgeSerializerV1', {})
+
+class SuperBadgeClassSerializerV1(serializers.Serializer):
+    name = StripTagsCharField(required=True, max_length=128)
+    # slug = StripTagsCharField(required=False, max_length=128, source='entity_id')
+    description = StripTagsCharField(required=False, allow_blank=True,
+            allow_null=True, max_length=255)
+    # badges = SuperBadgeBadgeSerializerV1(
+    #     read_only=False, many=True, required=False, source='cached_collects'
+    # )
+    badges = serializers.PrimaryKeyRelatedField(many=True, queryset=BadgeClass.objects.all(), required=False, source='cached_collects')
```

Comment:
A comment explaining the relationship of this "primary key related field" would be helpful